API.md

```diff
@@ -3394,6 +3394,18 @@ following options:
 - `noSniff` - boolean controlling the 'X-Content-Type-Options' header. Defaults to `true` setting
   the header to its only and default option, `'nosniff'`.
 
+- `referrer` - controls the ['Referrer-Policy'](https://www.w3.org/TR/referrer-policy/) header value:
+    - `false` - the 'Referrer-Policy' header will not be sent with responses. This is the default value.
+    - `''` - empty string indicating that the Referrer-Policy will be defined elsewhere.
+    - `'no-referrer'` - never include the referrer header.
+    - `'no-referrer-when-downgrade'` - never include the referrer when navigating from HTTPS to HTTP.
+    - `'same-origin'` - only include the referrer on the current site origin.
+    - `'origin'` - include the referrer but strip off path information so that the value is the current origin only.
+    - `'strict-origin'` - same as `'origin'` but the referrer will be omitted when going from HTTPS to HTTP.
+    - `'origin-when-cross-origin'` - the referrer includes the full path for same-origin requests but only the origin components of the URL are included for cross origin requests.
+    - `'strict-origin-when-cross-origin'` - same as `'origin-when-cross-origin'` but the referrer will be omitted when going from HTTPS to HTTP.
+    - `'unsafe-url'` - the referrer will always be included with the full URL.
+
```

Comment:
Basically, the descriptions are confusing because they are about the HTTP policy, not the hapi server actions. Needs to rewrite them to read "Informs the client to...".

Reply:
makes sense, I'll update